src/utils/object.ts

```diff
@@ -4,9 +4,9 @@
  * @author Andreas Geyer <andreas.geyer@t-systems.com>
  * @param object
  */
-export function recursiveObjectSort(object: object): object {
-    const sorted_object: object = {};
-    const sorted_objects: object[] = [];
+export function recursiveObjectSort(object: { [index: string]: any }): object {
```

Comment:
Replacing `object` with this construct makes clear that we're talking about a string-indexed object. 
Satisfies [noUncheckedIndexedAccess](https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess)

Reply:
I don't know. The key could be anything, not only string I guess.